torch/csrc/jit/frontend/source_range.cpp

```diff
@@ -106,6 +106,8 @@ C10_EXPORT void SourceRange::print_with_context(
   }
   AT_ASSERT(end_highlight == str.size() || str[end_highlight] == '\n');
 
+  out << "SourceRange at:"
```

Comment:
Why breaking this line?

Reply:
This is to follow the clang format. The code is automatically formatted by a script.